libretro.cpp

```diff
@@ -1715,12 +1732,13 @@ void retro_set_environment(retro_environment_t cb)
 
    static const struct retro_variable vars[] = {
       { "pce_fast_cdimagecache", "CD Image Cache (Restart); disabled|enabled" },
-      { "pce_nospritelimit", "No Sprite Limit; disabled|enabled" },
+      { "pce_nospritelimit", "No Sprite Limit (Restart); disabled|enabled" },
+      { "pce_hoverscan", "Horizontal Overscan (Restart); disabled|enabled" },
```

Comment:
I'm sure it's possible to make these settings work without a restart of the... core?  Not simply the game?  What does "restart" mean?

Anyway, having the setting is an improvement over no setting, so getting rid of the "restart" restriction can be a feature for later.